package.json

```diff
@@ -49,6 +49,7 @@
     "semantic-release": "^8.2.0"
   },
   "dependencies": {
+    "is-relative-path": "^1.0.0",
```

Comment:
This is the version that supports the behavior I'd like to enforce

| path      | v1.x     | v2.x     |
| --------- | -------- | -------- |
| `"" `     | false    | **true** |
| `"."`     | true     | true     |
| `".."`    | true     | true     |
| `"foo"`   | false    | **true** |
| `"/foo"`  | false    | false    |
- [Source](https://github.com/mrjoelkemp/is-relative-path/)